documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -174,6 +175,46 @@ A Property `errorMessages` with a Collection Characteristic and effective data t
 }
 ----
 
+Another one example for `samm-c:Either`, a Property `speedProperty` has Characteristic and effective data type `samm-c:Either` and the possible values
+`isMoving` (`xref:datatypes.adoc#data-types[xsd:boolean]` data type) or '60' (`Number`) would be serialized in the JSON payload as follows:
+[source,json]
+----
+{
+  "speedProperty": {
+    "left": true
+  }
+}
+
+or
```

Comment:
This 'or' is in the middle of the JSON source block. Instead the block before should be closed and the one after should be opened again